VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -180,12 +180,10 @@ private void tryToUpdateForEndOfSequential(){
             String nextUnitSubsectionId = unitList.get(curIndex + 1).getParent().getId();
```

Comment:
>// fix: https://openedx.atlassian.net/browse/MA-995
// code below decides to show Next/Previous Unit name or only Next/Previous
// based on units in a subsection

These comments should also be deleted.